README.md

```diff
@@ -1 +1,107 @@
-# peer-did-python
\ No newline at end of file
+# Peer DID python API
+
+## Information
```

Comment:
I would remove the Information header, so that general info is part of the main Peer DID python API header.